module5/preparation/Week1/ErrorHandling.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Error Handling Prework
+---
+
+## Pre-work for [Error Handling](/module5/lessons/Week1/ErrorHandling)
+
+When an application is running, there are all kinds of unexpected things that can and will happen. Maybe the API you are using goes down, or maybe the user inputs a string for a field that should be a number.
+
+It's our job as developers to identify, manage, and respond to these situations. We want to handle these cases gracefully and make sure the application doesn't crash and we give useful messages to the user about what went wrong.
+
+To prepare for the lesson on Error Handling, read [Writing Helpful Error Messages](https://developers.google.com/tech-writing/error-messages) from the Google Technical Writing Course to help you start thinking about what goes into good and bad error handling.
+
+Then read the following Error Handling Rules:
+* [Don't Fail Silently](https://developers.google.com/tech-writing/error-messages/error-handling#dont_fail_silently)
+* [Avoid Swallowing the Root Cause](https://developers.google.com/tech-writing/error-messages/error-handling#avoid_swallowing_the_root_cause)
+* [Log The Error Codes](https://developers.google.com/tech-writing/error-messages/error-handling#log_the_error_codes)
+* [Raise Errors Immediately](https://developers.google.com/tech-writing/error-messages/error-handling#raise_errors_immediately)
+
+
+✅ Deliverable: ???????????? TBD
```

Comment:
I wouldn't stress too much about a deliverable - as we get further in the program, this could fall under 'release of responsibility' 🙃

Reply:
I like that perspective